components/Code.js

```diff
@@ -10,6 +10,7 @@ const Code = styled.span`
   font-weight: 300;
   padding: 0 0.05em 0.1em;
   vertical-align: bottom;
+  line-height: 1.4;
```

Comment:
@ucorina @mxstbr Hiya, not sure if this was intentional? I think the alignment of the inline-code element took a while, so I'm not sure whether this will turn out correct, or whether this changes it at all

Reply:
> that's to keep the text in monospace font aligned with everything else in titles and content text.

Reply:
Cheers, must’ve overlooked that 😅 apologies